index.html

```diff
@@ -370,6 +370,27 @@ <h4>Multikey</h4>
   ]
 }
           </pre>
+
+          <p>
+            The `secretKeyMultibase` property represents a Multibase-encoded Multikey
+            expression of a P-256 or P-384 secret key (also sometimes referred to
+            as a private key). The encoding of a P-256 secret key is
+            the two-byte prefix `0x8626` (the varint expression of `0x1306`) followed
+            by the 32-byte secret key data.
+            The 34-byte value is then encoded using base58-btc (`z`) as the prefix.
+
+            The encoding of a P-384 secret key is the two-byte prefix `0x8726` (the varint
+            expression of `0x1307`) followed by the 48-byte secret key data.
+            The 50-byte value is then encoded using base58-btc (`z`) as the prefix. Any
+            other encodings MUST NOT be allowed.
+          </p>
+
+          <p class="advisement">
+            Developers are advised to not accidentally publish a representation of a secret
+            key, and to not export the `secretKeyMultibase` property by default, when serializing
+            key pairs to Multikey.
```

Comment:
```suggestion
            Developers are advised to prevent accidental publication of a representation of a secret
            key, and to not export the `secretKeyMultibase` property by default, when serializing
            key pairs as Multikey.
```